packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -186,8 +193,12 @@ contract DeployImplementationsOutput {
         DeployUtils.assertValidContractAddresses(addrs);
     }
 
-    function opsm() public view returns (OPStackManager) {
+    function opsm() public returns (OPStackManager) {
         DeployUtils.assertValidContractAddress(address(_opsm));
+        // We prank as the zero address due to the Proxy's `proxyCallIfNotAdmin` modifier.
+        // Pranking inside this function also means it can no longer be considered `view`.
+        vm.prank(address(0));
```

Comment:
Let's put the prank inside the `assertEIP1967Implementation` method so it's easier to use. You'll have to manually define `vm` as a constant within the library but that's ok

Reply:
https://github.com/ethereum-optimism/optimism/pull/11875/commits/a77e799e4e37389132993e8883cd14678ff78d23#diff-774808493f37225476f807a6244ec5554bdecb0d8edd30f6dd4ed737f1ee0e90R9